teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
```

Comment:
- [ ] Similar I said in the Student class, please 🙏🏽 remove this line because we don't need @specialization outside the class Teacher.